solutions/chapter_11/wdio.android.emulator.conf.ts

```diff
@@ -0,0 +1,21 @@
+import type { Options } from '@wdio/types'
+import {join} from 'node:path';
+import {config as sharedConfig} from './wdio.conf.js';
+
+export const config: Options.Testrunner = {
+  ...sharedConfig,
+  capabilities: [
+    {
+      platformName: 'Android',
+      'appium:automationName': 'UIAutomator2',
+      // Change this to the name of your emulator
+      'appium:deviceName': 'Pixel_7_Pro_Android_14_API_34',
+      // Change this to the version of your emulator
+      'appium:platformVersion': '14.0',
+      // Change this to the path/name of your app
+      'appium:app': join(process.cwd(), './apps/android.wdio.native.app.v1.0.8.apk'),
```

Comment:
Maybe I missed this but where do people get this app from?

Reply:
It's a few lines above

```log
4. Create an `apps` folder at your project's root and download the required Android or iOS app from [here](https://github.com/webdriverio/native-demo-app/releases).
5. Create a file named `wdio.android.emulator.conf.ts` in the root of your project with the following contents:
```